WcaOnRails/app/views/delegate_reports/_delegate_report.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="delegate-report">
   <div>
     <strong>Date</strong>
-    <%= wca_date_range(competition.start_date, competition.end_date) %>
+    <%= wca_date_range(competition.start_date, competition.end_date, locale: :en) %>
```

Comment:
This is interesting, is this code used when generating the delegate report email?